acceptance_tests/features/steps/events_emitted.py

```diff
@@ -181,3 +181,10 @@ def case_updated_emitted_with_correct_sensitve_data(context):
                             f'The emitted case, {emitted_case} does not match the case {context.case_id}')
     test_helper.assertEqual(emitted_case["sampleSensitive"]['PHONE_NUMBER'], "REDACTED",
                             "Expected emitted_case['sampleSensitive']['PHONE_NUMBER'] to equal 'REDACTED'")
+
+
+@step("A UAC_UPDATE event is sent for the record in the sample file")
```

Comment:
petty but every thing else starts lowercase (except nouns like UAC_UPDATE) in steps, so
```suggestion
@step("a UAC_UPDATE event is sent for the record in the sample file")
```